prowler/lib/outputs/ocsf/ocsf.py

```diff
@@ -129,26 +151,19 @@ def transform(self, findings: List[Finding]) -> None:
                         "compliance": finding.compliance,
                     },
                 )
-
-                if finding.provider == "kubernetes":
-                    detection_finding.container = Container(
-                        name=finding.resource_name,
-                        uid=finding.resource_uid,
-                    )
-                    # TODO: Get the PID of the namespace (we only have the name of the namespace)
-                    # detection_finding.namespace_pid=,
-                else:
+                if finding.provider != "kubernetes":
                     detection_finding.cloud = Cloud(
                         account=Account(
                             name=finding.account_name,
                             type_id=cloud_account_type.value,
-                            type=cloud_account_type.name,
+                            type=cloud_account_type.name.replace("_", " "),
                             uid=finding.account_uid,
                             labels=unroll_dict_to_list(finding.account_tags),
                         ),
                         org=Organization(
                             uid=finding.account_organization_uid,
                             name=finding.account_organization_name,
+                            # TODO: add the org unit id and name
```

Comment:
Are we going to implement it in this PR?

Reply:
Nope.